po/pl.po

```diff
@@ -463,15 +463,15 @@ msgstr ""
 
 #: src/login/org.freedesktop.login1.policy:278
 msgid "Suspend the system while an application is inhibiting this"
-msgstr "Uśpienie systemu, kiedy program zażądał jego wstrzymania"
+msgstr "Uśpienie systemu, kiedy program zażądał wstrzymuje to"
 
 #: src/login/org.freedesktop.login1.policy:279
 msgid ""
 "Authentication is required to suspend the system while an application "
 "is inhibiting this."
 msgstr ""
-"Wymagane jest uwierzytelnienie, aby uśpić system, kiedy program zażądał jego "
-"wstrzymania."
+"Wymagane jest uwierzytelnienie, aby uśpić system, kiedy program wstrzymuje
```

Comment:
"Wymagane jest uwierzytelnienie, aby uśpić system, kiedy program je wstrzymuje"

Reply:
"to wstrzymuje" ?

Reply:
That's even better.